view/adminhtml/templates/form/payment-link.phtml

```diff
@@ -23,16 +23,35 @@ $url = $block->getUrl('rvvup/availability/index');
             Will create and share in order email
         </label>
     </div>
-    <script>
-        require(['jquery', 'Rvvup_Payments/js/available-payment'], function($, availablePayment) {
+    <?php if ($secureRenderer): ?>
```

Comment:
Secure renderer is available for >2.4.0, so we can don't need to duplicate the logic.